### ISSUE TITLE
Searching `.` in the command palette matches `:`, searching for multiple `.` shifts the words around

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode#153904]
Not sure if `.` does something special that I'm not aware of, but I was expected to filter to all commands ending in `...` here:

![Recording 2022-07-01 at 04 22 08](https://user-images.githubusercontent.com/2193314/176885188-dd2c9b5a-cfe0-4307-9bea-dd2e53644b56.gif)

**Comment by TylerLeonhardt** [comment]
Yeah we normalize certain characters and treat them as spaces... `.` being one of them. So `.` is the same as `:` being the same as a space.

https://github.com/microsoft/vscode/blob/71b996eff4cef4405d1d1cc4fe308545dac0b255/src/vs/base/common/filters.ts#L129

Perhaps removing `.` makes sense...

cc @bpasero

**Comment by Tyriar** [comment]
That makes sense I guess, one little thing that you may want to fix is how the text changes rendering when you type in the `...`, it's fairly obvious on a 100 device pixel ratio display. I'm assuming this is because the a bold space ends increasing the width of the space by < 1px. The fix could be not to apply bold to spaces the very start or end of the bold section.
